apps/src/codebridge/ControlButtons/control-buttons.module.scss

```diff
@@ -3,16 +3,19 @@
 .controlButtonsContainer {
   grid-area: control-buttons;
   display: grid;
-  grid-template-columns: 1fr repeat(2, auto) 1fr;
+  grid-template-columns: 1fr auto 1fr;
   padding: 8px 0;
   background-color: $neutral_dark80;
-  column-gap: 8px;
 }
 
-.firstControlButton {
+.centerButton {
   grid-column-start: 2;
 }
 
+.runButton {
+  margin-right: 8px;
+}
```

Comment:
zero important just curious: what trouble was `column-gap` causing?

Reply:
I changed this because I switched from having 1 column per button to having one column for run/test and one for continue. The run/test column is replaced by the stop button when we switch states. After moving to one column, I needed a margin on the run button so. the buttons weren't squished together.